README.md

```diff
@@ -34,20 +42,75 @@ verification-methods:
     - "/cert1.pem"
     - "https://www.example.com/cert2.pem" 
 ```
+The following attribute MUST be set:
+- `issuer`:  Issuer of DID document, which is the DID itself.
+- `verification-methods` List of public keys used as verification methods in DID document to be generated. scs-did-creator sets JWK as verification method. JWK is formatted according to [RFC 7517](https://datatracker.ietf.org/doc/html/rfc7517#section-4)) as verification method only. s At least one verification method MUST be set:
+  - `key`: Absolute file path to private key file. Using this setting adds JWK as verification method expressed by parameters `n` and `e`, only
```

Comment:
```suggestion
  - `keys` entry: Absolute file path to private key file. Using this setting adds JWK as verification method expressed by parameters `n` and `e`, only
```

I think we should reference the config format syntax more verbatim here. Otherwise users might be confused between `keys`/`x509s` in the config template/examples and the `key`/`x509` (without `s`) documented here.
Same goes for the next entry below.